tests/model/test_model.py

```diff
@@ -2207,6 +2249,40 @@ def set_from_list_of_dict(data):
         update_left_panel.assert_called_once_with()
         model.controller.update_screen.assert_called_once_with()
 
+    @pytest.mark.parametrize(
+        "event, final_desktop_notified_streams, ",
+        [
+            (
+                {
+                    "property": "desktop_notifications",
+                    "op": "update",
+                    "stream_id": 19,
+                    "value": False,
+                },
+                {15},
+            ),
+            (
+                {
+                    "property": "desktop_notifications",
+                    "op": "update",
+                    "stream_id": 30,
+                    "value": True,
+                },
+                {15, 19, 30},
+            ),
+        ],
+        ids=["remove_19", "add_30"],
+    )
+    def test__handle_subscription_event_desktop_notified_streams(
+        self, model, mocker, stream_button, event, final_desktop_notified_streams
+    ):
+        event["type"] = "subscription"
+        model.desktop_notified_streams = {15, 19}
```

Comment:
I feel like should be in the parameterize blocks as reading `final_desktop_notified_streams` in the parameterization doesn't tell how we reached there finally.